### ISSUE TITLE
Barb Camp Cleared message don't match on money

### ISSUE THREAD

**Issue by saamohod** [LoneGazebo/Community-Patch-DLL#10980]
_1. Mod Version (X.Y.Z). Current Version: 4.8.0_

_4. Describe the Issue_
![20240612214916_1](https://github.com/LoneGazebo/Community-Patch-DLL/assets/16341598/81ecfb80-2040-47f0-92df-76f4b98bd9aa)
![20240612214920_1](https://github.com/LoneGazebo/Community-Patch-DLL/assets/16341598/b9c9d8c8-1d87-4c8e-af77-b26029bc9c81)

**Comment by Hokath** [comment]
Suspicious.
What speed are you playing at, Epic?
And which number is actually correct? Did you get the 225 or the 150?

**Comment by saamohod** [comment]
> Suspicious. What speed are you playing at, Epic? And which number is actually correct? Did you get the 225 or the 150?

Yes. It's one step slower than normal. Marathon or Epic; I always confuse them.
I never checked which number is actually the correct one. When it happens next time I'll try to verify.
